addons/website/static/src/interactions/image_shape_hover_effect.js

```diff
@@ -88,6 +100,9 @@ export class ImageShapeHoverEffect extends Interaction {
      * @param {Function} resolve
 ￼    */
     setImgSrc(svg, resolve) {
+        if (this.isDestroyed) {
+            return;
+        }
```

Comment:
Why do you need it finally ?

Reply:
Because protectSyncAfterAsync does not check for `this.isDestroyed`